default_multi_user/tests/test_default_multi_user.py

```diff
@@ -3,58 +3,59 @@
 
 import json
 
-from odoo.tests import common
+from odoo.tests import tagged
+from odoo.tests.common import TransactionCase
 
 
-class TestDefaultMultiUser(common.SavepointCase):
-    @classmethod
-    def setUpClass(cls):
-        super().setUpClass()
-        cls.env = cls.env(
-            context=dict(cls.env.context, tracking_disable=True, no_reset_password=True)
+@tagged("post_install", "-at_install")
+class TestDefaultMultiUser(TransactionCase):
```

Comment:
Any spacial reason to change to TransactionCase?

Reply:
@LoisRForgeFlow I've changed it to SavepointCase again. No need to convert it into TransactionCase so that all the test cases can use the same data without having to recreate test data and also use of setUpClass() is correct to create DB test data once.
Thanks!